index.html

```diff
@@ -133,18 +133,21 @@ <h6 class="title" id="about-title">About Me</h6>
   <section id="form-container">
     <h6 id="contact-title">CONTACT US</h6>
     <p class="blurb" id="form-blurb">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it.</p>
-    <form name="form" id="form" action="https://formspree.io/f/mbjblgra" method="POST">
+    <form name="form" id="form" onsubmit="return validator(this)" action="https://formspree.io/f/mbjblgra" method="POST">
       <div class="input">
         <label class="label" for="name">Name</label>
         <input class="input-field" type="text" id="name" name="name" placeholder="Full name" maxlength="30" required>
+        <small></small>
       </div>
       <div class="input">
         <label class="label" for="email">Email</label>
-        <input class="input-field" type="email" id="email" name="email" placeholder="Email address" required>
+        <input class="input-field" type="email" id="email" name="email" placeholder="Email address">
+        <small id="emailerror"></small>
```

Comment:
- [ ] Kindly notice that the error message should show up close to the "submit" button.